plugin/pulpcore/plugin/changeset/iterator.py

```diff
@@ -93,7 +93,8 @@ def _collated_content(self):
             dict: A dictionary of {model_class: [content,]}
                 Each content is: PendingContent.
         """
-        for batch in BatchIterator(self.content, 1024):
+        # higher values may hit the sqlite expression tree size limit
```

Comment:
Suggest: `should be < 1000 due to sqlite limitation. ` That way if someone changes the value later the comment is still accurate and relevant.

Reply:
+1